aspnetcore/fundamentals/app-state.md

```diff
@@ -12,7 +12,7 @@ uid: fundamentals/app-state
 
```

Comment:
I'll fix this in #17122
back this out now so I don't get a merge conflict.

Reply:
oh, ok ... will do.